packages/components/src/components/mobile-drawer/mobile-drawer-submenu.tsx

```diff
@@ -23,11 +24,12 @@ const SubMenu = ({
     submenu_title,
     submenu_suffix_icon,
     route_config_path,
+    is_expanded = false,
 }: React.PropsWithChildren<TMobileDrawerSubmenu>) => {
-    const [is_expanded, setIsExpanded] = React.useState(false);
+    const [is_extended, setIsExpanded] = React.useState(is_expanded);
```

Comment:
can we rename this `setIsExpanded` to `setIsExtended`

Reply:
yes, sure, thanks for noticing it